charts/flagsmith/templates/deployment-task-processor.yaml

```
@@ -104,5 +104,9 @@ spec:
 {{ toYaml .Values.taskProcessor.resources | indent 10 }}
 {{- with .Values.taskProcessor.extraContainers }}
 {{- toYaml . | nindent 6 }}
+{{- end }}
+      volumes:
+{{- with .Values.api.extraVolumes }}
```

Hmm... this one is interesting. I'm not sure if this is intentional or a typo (perhaps the former given that we have defined `taskProcessor.extraVolumes` in values.yaml below. 

If it's relevant, my preference is to be explicit here and have people duplicate the volume definitions if necessary. I know we have the `extraEnv` which is shared by api and task processor, but tbh, I'd rather we didn't! 

```suggestion
{{- with .Values.taskProcessor.extraVolumes }}
```

I'm open to discussion here though.

---

This was a genuine typo. Thanks for the catch!